tools/cellprofiler/test-data/common_image_math.cppipe

```diff
@@ -0,0 +1,94 @@
+CellProfiler Pipeline: http://www.cellprofiler.org
```

Comment:
is this how a file always needs to look like?

Reply:
I guess so... but not sure, maybe @sunyi000 knows :)